includes/models/model.llms.student.php

```diff
@@ -1841,4 +1909,162 @@ private function update_completion_status( $status, $object_id, $object_type, $t
 
 	}
 
+	/**
+	 * Determine if the student has favorited a lesson.
+	 *
+	 * @since [version]
+	 *
+	 * @param int    $object_id   WP Post ID of a lesson.
+	 * @param string $object_type Type, 'Lesson'.
```

Comment:
just a reminder of what said elsewhere about the object type description.

Reply:
fixed at all places